inc/cachify.class.php

```diff
@@ -641,7 +641,7 @@ public static function add_dashboard_count( $items = array() )
 
 		/* Right now item */
 		$items[] = sprintf(
-			'<a href="%s" class="cachify-icon cachify-icon--%s" title="%s: %s">%s esc_html__( 'Cache', 'cachify' )</a>',
```

Comment:
Maybe we should keep it translateable. But I would prefer to move it to the `%s` instead of the string itself. Something like:
```
( empty($size) ? esc_html__( 'Empty Cache', 'cachify' ) : size_format($size) . ' ' . esc_html__( 'Cache', 'cachify' ) )
```